application/app/components/content/accordion-actions.tsx

```diff
@@ -0,0 +1,183 @@
+			environnemental minime, pour développer les énergies renouvelables sur votre territoire.
+			Ces installations présentent également des vertus pédagogiques pour sensibiliser et
+			former les générations futures aux enjeux de transition énergétique.
+		</p>
+		<p>
+			C’est aussi, une fois l’investissement remboursé, une opportunité de réduire les
+			factures énergétiques de la commune tout en réduisant son empreinte carbone !
+		</p>
+		<br />
+		<p>
+			À Greenpeace France, nous pensons que pour réussir la transition énergétique des
+			bâtiments publics, ces projets solaires doivent s’inscrire dans une réflexion plus
+			générale sur l’état du bâti. Pour les bâtiments passoires énergétiques il est
+			indispensable de les rénover de façon performante avant d’y installer des panneaux
+			solaires.
+		</p>
+		<br />
+	</>
+);
+
+export const ELU_BODY = (
+	<ul className='list-inside list-disc'>
+		<li className='mb-8 mt-2'>
+			De nombreuses communes nous ont fait remonter leurs difficultés à financer ces projets,
+			de rénovation comme de panneaux photovoltaïques, par manque de moyens financiers mis par
+			l’État.
+			<span className='font-bold'>
+				&nbsp;Si cela est votre cas, nous vous invitons à nous contacter à l’adresse
+				suivante
+			</span>
+			<a
+				href={ACTION_LINKS.contact.url}
+				className='mt-3 block w-full rounded-md bg-green px-4 py-2 text-center font-bold text-darkgreen hover:underline'
+			>
+				{ACTION_LINKS.contact.label}
+			</a>
+		</li>
+		<li className='mb-10'>
+			De multiples démarches sont nécessaires pour monter un tel projet : conformité avec le
+			PLU(i) et si dans zone protégée avis des ABF, étude de faisabilité, modèle économique,
+			plan de financement, demande de subventions, demande de raccordement au réseau, …
+			<br />
+			<span className='font-bold'>
+				Voici quelques ressources pour vous aider dans l’élaboration de ces projets :
+			</span>
+			<br />
+			<p className='mt-3'>
+				– Le site{' '}
+				<a
+					href='https://www.photovoltaique.info'
+					target='_blank'
+					rel='noopener noreferrer'
+					className='text-green underline decoration-dotted decoration-2 underline-offset-4'
+				>
+					photovoltaïque.info
+				</a>{' '}
+				de l’association Hespul
+			</p>
+			<p>
+				– Le guide{' '}
+				<a
+					href='https://www.amorce.asso.fr/publications/elu-et-photovoltaique'
+					target='_blank'
+					rel='noopener noreferrer'
+					className='text-green underline decoration-dotted decoration-2 underline-offset-4'
+				>
+					L’ÉLU et le photovoltaïque
+				</a>
+			</p>
+			<p>
+				– Le guide{' '}
+				<a
+					href='https://www.banquedesterritoires.fr/energies-renouvelables-guide-collectivites'
+					target='_blank'
+					rel='noopener noreferrer'
+					className='text-green underline decoration-dotted decoration-2 underline-offset-4'
+				>
+					Mieux maîtriser le développement des EnR sur son territoire
+				</a>
```

Comment:
Il manque la fin de phrase et le lien n'est pas le bon, attention comme ceci est un PDF, en terme d'éco-conception il serait intéressant de fournir le format + la taille du fichier (au moins entre parenthèse) pour informer l'utilisateur et éviter des échanges de données superflus.
Taille = 1.6mo

```suggestion
				– Le guide{' '}
				<a
					href='https://energie-partagee.org/wp-content/uploads/2020/12/BDT-2020-Guide-collectivite%CC%81s-Mieux-maitriser-de%CC%81veloppement-EnR-sur-territoire.pdf'
					target='_blank'
					rel='noopener noreferrer'
					className='text-green underline decoration-dotted decoration-2 underline-offset-4'
				>
					Mieux maîtriser le développement des EnR sur son territoire
				</a>
				de la Banque des Territoires
```